package.json

```diff
@@ -112,6 +112,7 @@
   },
   "dependencies": {
     "@babel/core": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
```

Comment:
Installing this dependency caused no changes in the yarn.lock file. I didn't forget to commit it. 😉

Reply:
Oh, I just pulled down to verify 😄